app/controllers/v0/post911_gi_bill_statuses_controller.rb

```diff
@@ -1,25 +1,42 @@
 # frozen_string_literal: true
+require 'evss/gi_bill_status/gi_bill_status_response'
+
 module V0
   class Post911GIBillStatusesController < ApplicationController
     include SentryLogging
 
+    STATSD_GI_BILL_TOTAL_KEY = 'api.evss.gi_bill_status.total'
+    STATSD_GI_BILL_FAIL_KEY = 'api.evss.gi_bill_status.fail'
+
     def show
       response = service.get_gi_bill_status
-      if response.contains_education_info?
-        # 200
+      if response.success?
         render json: response,
                serializer: Post911GIBillStatusSerializer,
                meta: response.metadata
-      elsif response.evss_error?
+      else
+        error_type = response.error_type
+        StatsD.increment(STATSD_GI_BILL_FAIL_KEY, tags: ["error:#{error_type}"])
+        render_error_json(error_type)
+      end
+    ensure
+      StatsD.increment(STATSD_GI_BILL_TOTAL_KEY)
+    end
+
+    private
+
+    def render_error_json(error_type)
```

Comment:
I also think that this is debt and goes against 'skinny controllers', but I'm not sold that the solution is to raise from the service.  It's the responsibility of the Controller (in my own words) to interpret HTTP requests and turn them into HTTP responses.  Raising from the service kind of takes the "HTTP-response" part of that responsibility away from the controller.  It also takes some response parsing responsibility away from the `GiBillStatusResponse` class.

This also is a product of our `raise Exception` is effectively equivalent to `render json:` functionality, which is a little wonky.

My vote is to take on the debt for now and log an issue to design and fix later.